### PR TITLE
Fix mem_info readings to be more reliable

### LINKED ISSUE #1127: mem_info not reliable
The human friendly representation gives integer representation instead of float which gives unreliable stats.
```
>>> from circus.util import bytes2human
>>> bytes2human(1024 * 1024 * 2047)
'1G'
>>> bytes2human(1024 * 1024 * 2048)
'2G'
```

The first example should return `1.99G` ideally. This is not critical when talking about megas but when we start talking about gigas or more it gets problematic.

Also why not always give a MiB representation in float (not string) ?

### PR DESCRIPTION
**Pull request by biozz**
This fixes #1127.

- add conversion to string and cutting to 2 decimal places
- add tests for edge cases with 1.99G and 2.00G

### CODE REVIEW

File: circus/util.py [hunk 1]
```diff
@@ -144,7 +144,12 @@ def bytes2human(n):
 
     for s in reversed(_SYMBOLS):
         if n >= prefix[s]:
-            value = int(float(n) / prefix[s])
+            # truncate to 2 decimal places without rounding
+            # https://stackoverflow.com/questions/783897/truncating-floats-in-python
```
Comment (k4nar) [inline circus/util.py]: What would be wrong with rounding though? (`round(float(n) / prefix[s], 2)`)

File: circus/tests/test_util.py [hunk 2]
```diff
@@ -92,8 +92,11 @@ def test_convert_opt(self):
         self.assertEqual(util.convert_opt('test', 1), '1')
 
     def test_bytes2human(self):
-        self.assertEqual(bytes2human(10000), '9K')
-        self.assertEqual(bytes2human(100001221), '95M')
+        self.assertEqual(bytes2human(100), '100B')
+        self.assertEqual(bytes2human(10000), '9.76K')
```
Comment (k4nar) [inline circus/tests/test_util.py]: Ideally this should be `9.76 KiB`, but I'm probably being too pedantic here ;) .

File: circus/util.py (hunk 1 above)
Comment (biozz) [reply circus/util.py]: `round(..., 2)` for `1024 * 1024 * 2047` will give `2.00G`, but it is in fact `1.99G`.

Comment (k4nar) [reply circus/util.py]: I don't see what's wrong with `2.00G` in this case. In the original issue we were displaying `1G`, which was plain wrong. But here `2.00G` is the right rounding IMO.

File: circus/tests/test_util.py (hunk 2 above)
Comment (biozz) [reply circus/tests/test_util.py]: KiB notation is kind of controversial, despite being a standard. It is not as widely adopted in my opinion as the one used right now.

Anyway, if you think this is the right way I can do that in a separate PR.

File: circus/util.py (hunk 1 above)
Comment (biozz) [reply circus/util.py]: Ok, no problem. I was just trying to reproduce the example from the issue.

I changed to round, fixed tests and also made readings fixed width, because it was printing `2.0G` after rounding and not `2.00G` as with all other values.

File: circus/tests/test_util.py (hunk 2 above)
Comment (k4nar) [reply circus/tests/test_util.py]: Let's keep `K` for simplicity for now.
